src/test_only_plugins/anonymizeip.py

```diff
@@ -1,126 +0,0 @@
-# Port of https://github.com/samuelmeuli/anonymize-ip to Python 2
```

Comment:
Can you bring this file back, under `new_sentry_plugins`? Looks like it's used by `src/new_sentry_plugins/splunk/plugin.py`.